parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -71,6 +71,16 @@ $global:CNIConfig = [Io.path]::Combine($global:CNIPath, "config", "`$global:Netw
 $global:HNSModule = [Io.path]::Combine("$global:KubeDir", "hns.psm1")
 
 $global:VolumePluginDir = [Io.path]::Combine("$global:KubeDir", "volumeplugins")
+#azure cni
+$global:NetworkPolicy = "{{WrapAsVariable "networkPolicy"}}"
+$global:VNetCNIPluginsURL = "{{WrapAsVariable "vnetCniWindowsPluginsURL"}}"

```

Comment:
Where is vnetCniWindowsPluginsURL defined?

Reply:
In parts/k8s/kubernetesparams.t